pkg/webview/view.proto

```diff
@@ -14,7 +14,8 @@ message BuildRecord {
   repeated string warnings = 3;
   google.protobuf.Timestamp start_time = 4;
   google.protobuf.Timestamp finish_time  = 5;
-  string log = 6;
+  reserved "log";
```

Comment:
today i learned: this is the proto3 way to mark a field as deleted
https://developers.google.com/protocol-buffers/docs/proto3#reserved